defold-spine/src/gui_node_spine.cpp

```diff
@@ -36,7 +37,7 @@ static const dmhash_t SPINE_SCENE_SUFFIX        = dmHashString64(".spinescenec")
 
 struct GuiNodeTypeContext
 {
-    // In case we need something later. Here for visibility
+    spSkeletonClipping* m_SkeletonClipper;
```

Comment:
This actually allocates a bit of memory, so we typically don't want to allocate on of these per node. We just need one of these temporarily when generating the vertices